src/components/alerts/MemberValidationBanner.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import {
+  Alert,
+  AlertTitle,
+  IconButton,
+  Link as MUILink,
+  Stack,
+} from '@mui/material';
+
+import { XIcon } from 'lucide-react';
+
+import { useBuilderTranslation } from '@/config/i18n';
+import { hooks } from '@/config/queryClient';
+import {
+  MEMBER_VALIDATION_BANNER_CLOSE_BUTTON_ID,
+  MEMBER_VALIDATION_BANNER_ID,
+} from '@/config/selectors';
+import { BUILDER } from '@/langs/constants';
+
+const DOCUMENTATION_ORIGIN = 'https://graasp.github.io/docs';
+const MEMBER_VALIDATION_DOCUMENTATION_LINK = '/user/account/validation';
+const buildLocalizedDocumentationOrigin = (lang: string = 'en') => {
+  switch (lang) {
+    case 'en':
+      return DOCUMENTATION_ORIGIN;
+    default:
+      return `${DOCUMENTATION_ORIGIN}/${lang}`;
+  }
+};
+// eslint-disable-next-line arrow-body-style
+const buildLocalizedDocumentationLink = (lang: string): string => {
+  // english does not use a path prefix
+  return `${buildLocalizedDocumentationOrigin(lang)}${MEMBER_VALIDATION_DOCUMENTATION_LINK}`;
+};
+
+const MemberValidationBanner = (): JSX.Element | false => {
+  const [open, setOpen] = useState(true);
```

Comment:
If you want, I made a really small hook that allow you to simplify the modal's state ^^. It's not a big improvements so it's really up to you ;)

```suggestion
  const { isOpen, openModal, closeModal } = useModalStatus({ isInitiallyOpen: true });
```